sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryAvroUtils.java

```diff
@@ -47,6 +47,19 @@
  */
 class BigQueryAvroUtils {
 
+  public static final ImmutableMap<String, Type> BIG_QUERY_TO_AVRO_TYPES =
```

Comment:
Yes.

The fakes and mocks in BigQueryIOTest and BigTableIOTest weren't set up enough to handle the queries made when initial splitting is exercised, which meant the tests failed.

I initially manually constructed an Avro schema, but this seems more effective.